flask_resty/testing.py

```diff
@@ -1,12 +1,36 @@
+from datetime import datetime, timedelta
+import re
+
+from dateutil import parser
```

Comment:
i feel like it's a bit weird to introduce dateutil as a dependency here when it's only used for test support. maybe we can leave out the date assertions?